modules/create-crio-default-capabilities.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
I see that this is listed as a procedure and the module title starts with a gerund, also indicating that this is a procedure. However, I don't see any steps. Can some of the paragraphs above the NOTE be restructured into steps?

Reply:
Can I change it as CONCEPT? Initially when I started working it did have steps but those steps were no more relevant so it was restructured as the present module.

Reply:
@subhtk Yes, I think it's acceptable to change it to a CONCEPT.

Reply:
Great. Thanks